src/vl-cookie-consent.js

```diff
@@ -426,10 +351,16 @@ export class VlCookieConsent extends VlElement(HTMLElement) {
             }
         }
     }
-}
 
+    _ownerChangedCallback(oldValue, newValue) {
+        this._ownerElement.innerText = newValue;
+    }
 
-define('vl-cookie-consent-opt-in', VlCookieConsentOptIn);
+    _linkChangedCallback(oldValue, newValue) {
+        this._linkElement.innerText = newValue;
+        this._linkElement.href = newValue;
```

Comment:
zeker dat ze hiertussen niet nog eens onderscheid nodig hebben?

Reply:
Dat was nu niet het geval, dat kan toegevoegd worden. Is eerder een functionele vraag voor @kevcraey.

Reply:
Nee onderscheid link en de tekst van de link moet niet gemaakt kunnen worden.